generator-mediator-js/app/templates/mediatorTemplate/lib/_index.js

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env node
+'use strict'
+
+const express = require('express')
+const medUtils = require('openhim-mediator-utils')
+
+const utils = require('./utils')
+
+// Config
+var config = {} // this will vary depending on whats set in openhim-core
+const apiConf = process.env.NODE_ENV === 'test' ? require('../config/test') : require('../config/config')
+const mediatorConfig = require('../config/mediator')
+
+var port = process.env.NODE_ENV === 'test' ? 7001 : mediatorConfig.endpoints[0].port
+
+/**
+ * setupApp - configures the http server for this mediator
+ *
+ * @return {express.App}  the configured http server
+ */
+function setupApp () {
+  const app = express()
+
+  app.all('*', (req, res) => {
+    console.log(`Processing ${req.method} request on ${mediatorConfig.endpoints[0].path}`)
```

Comment:
mHero's already got a setup that you can use: https://github.com/jembi/openhim-mediator-mhero/blob/master/index.js#L27-L29